ui/raidboss/data/02-arr/raid/t10.js

```diff
@@ -91,7 +91,6 @@
       'locale': 'de',
       'replaceSync': {
         'Imdugud': 'Imdugud',
-        'The Alpha Concourse': 'Alpha-Emergenzzone',
```

Comment:
Why are you removing this? https://github.com/quisquous/cactbot/blob/master/ui/raidboss/data/02-arr/raid/t10.txt#L12

Reply:
On the Alex raid, there are these lines also in the files (txt), but they are not written in the files (js) on (replaceSync).  I did not check for the other raids.
- Either it is useless because the check is ok. 
- Or it is an oversight on the alex files.

Isn't it just useful to sync the bosses? Tell me what to do and I'll do it ^^.

Reply:
I think this is an oversight on the alex files.  Many timelines have extra syncs in case there is not a translation (so that it will still start), but having the zone translation will allow the timeline to start immediately.

Reply:
All right, I will correct that.